DEV/org.openl.rules.util/src/org/openl/rules/util/Round.java

```diff
@@ -28,13 +28,20 @@ private Round() {
     /**
      * Like {@link #round(double)} but null-safe.
      */
-    public static Long round(Double value) {
+    public static Integer round(Double value) {
         if (value == null) {
             return null;
         }
         return round((double) value);
     }
 
+    public static Integer round(Double value, RoundingMode roundingMode) {

```

Comment:
It needs to research the way, how to define rounding mode without extra imports. OpenL functions should not have dependencies on anything outside of java.lang package, So, maybe it needs to introduce own RoundingMode type inside this class.